### DISCUSSION TITLE
Plus migration to Qt6

### DISCUSSION THREAD

**Discussion by atracsys-sbt** [PlusToolkit/PlusLib#1046] [General]
Hi,
I was just wondering if the migration of PlusApp to Qt6 was in the works/coming soon or if it's not something we should expect for some time.
Thanks

**Comment by lassoan** [comment]
Normally we would update Plus to use Qt6 when 3D Slicer switches to use Qt6. 3D Slicer will switch when all dependencies support Qt6, specifically:
- VTK ✔️
- PythonQt - https://github.com/MeVisLab/pythonqt/issues/13
- CTK - https://github.com/commontk/CTK/issues/947

I would estimate that it will not happen within 6 months, but probably will not take more than a year.

**Reply by atracsys-sbt** [reply]
Thank you for your response.
